src/main/java/com/zontik/groshiky/model/TransactionMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.zontik.groshiky.model;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Objects;
+
+@Component
+public class TransactionMapper {
+
+    @Autowired
+    private ModelMapper mapper;
+
+    public Transaction toEntity(TransactionDto dto) {
+        return Objects.isNull(dto) ? null : mapper.map(dto, Transaction.class);
```

Comment:
In this case your mapper will map only fields with similar types and names. You could add to this modelMapper different mappings to map, for example, user id to User (I mean dto contains user id and model mapper makes User from provided id)